src/current/_data/versions.csv

```diff
@@ -16,4 +16,4 @@ v23.2,2024-02-05,2025-02-05,2025-08-05,23.2.6,23.2.7,2024-07-08,2025-07-08,2026-
 v24.1,2024-05-20,2025-05-20,2025-11-20,24.1.5,24.1.6,2024-10-21,2025-10-21,2026-10-21,v23.2,release-24.1
 v24.2,2024-08-12,2025-02-12,N/A,N/A,N/A,N/A,N/A,N/A,v24.1,release-24.2
 v24.3,2024-11-18,2025-11-18,2026-05-18,N/A,N/A,N/A,N/A,N/A,v24.2,release-24.3
-v25.1,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.2,release-24.3

```

Comment:
I think this change should be reverted -- see https://cockroachlabs.slack.com/archives/C0306LF0W69/p1736285588302159 for context

Reply:
DMed you about this